tool/python/examples/train_cifar10.py

```diff
@@ -28,27 +28,49 @@
 import numpy as np
 
 current_path_ = os.path.dirname(__file__)
-singa_root_=os.path.abspath(os.path.join(current_path_,'../../..'))
+singa_root_ = os.path.abspath(os.path.join(current_path_,'../../..'))
 sys.path.append(os.path.join(singa_root_,'tool','python'))
 
 from singa.driver import Driver
 from singa.layer import *
 from singa.model import *
-from singa.utils.utility import swap32
 
-fname_mean_image = 'tool/python/examples/datasets/cifar10_mean_image'
-mean_image = np.fromfile(fname_mean_image)
+dataset_dir_ = singa_root_ + "/examples/cifar10/cifar-10-batches-bin"
+mean_image = None
+
+def compute_mean_image():
+    ''' This is a sample script to cmopute the average image
+        of all samples in 5 dataset of cifar10
+        See [https://www.cs.toronto.edu/~kriz/cifar.html] for more details
+    '''
+    nb_samples = 10000
+    nb_pixels = 3 * 32 * 32  
+    mean = None
+    nb_samples_total = 0
+    for did in range(1,6):
+        fname_train_data = dataset_dir_ + "/data_batch_{}.bin".format(did)
+        d = np.fromfile(fname_train_data, dtype=np.uint8)
+        d = d.reshape(nb_samples, nb_pixels + 1) # +1 for label
+        num = d.shape[0]
+        img = d[:, 1:] 
+        img = img.mean(axis=0)
```

Comment:
img's dtype is unint8, then the mean is uint8 (not float).
would there be any effect on the accuracy?